phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -167,9 +167,9 @@ public static synchronized void setup(
             //Maybe we're gonna have to refactor the namespace
             if(clusterInitialized) {
                 tearDownMiniCluster(0);
-                System.setProperty("java.io.tmpdir", tmpDir);
+                System.setProperty("java.io.tmpdir", tmpPath);
```

Comment:
The old variable name is better here.
This is not used a base for paths, it's only used for saving the value of the system property.

Reply:
and the old variable name reflects this better.